plugin/trino-blackhole/src/test/java/io/trino/plugin/blackhole/TestBlackHoleSmoke.java

```diff
@@ -75,8 +75,8 @@ public void testCreateSchema()
         assertEquals(queryRunner.execute("SHOW SCHEMAS FROM blackhole").getRowCount(), 3);
         assertThatQueryReturnsValue("CREATE TABLE test.test_schema as SELECT * FROM tpch.tiny.region", 5L);
 
-        assertThatQueryReturnsValue("DROP TABLE test_schema", true);
-        assertThatQueryReturnsValue("DROP TABLE test.test_schema", true);
+        assertThatQueryDoesntReturnValue("DROP TABLE test_schema");
```

Comment:
Contractions do not look well in method names, since the apostrophe is missing.
spell out "..DoesNot..."